integration-deploy-service/action.yml

```diff
@@ -0,0 +1,58 @@
+name: "Integration Deploy Service"
+description: "Deploy Service using Maven"
+
+inputs:
+  AWS_ACCESS_KEY_ID:
+    description: "AWS access key used to write to ECR"
+    required: true
+  AWS_SECRET_ACCESS_KEY:
+    description: "AWS secret access key used to write to ECR"
+    required: true
+  AWS_REGION:
+    description: "AWS region"
+    required: true
+  AWS_PROFILE:
+    description: "AWS profile"
+    required: true
+  AWS_CONFIG:
+    description: "AWS configuration"
+    required: true
+  MVN_REPO_TOKEN:
+    description: "Token for private Maven repo"
+    required: true
+  NAME:
+    description: "Service name"
+    required: true
+  CLUSTER_NAME:
+    description: "Cluster name"
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: Write AWS config
+      shell: bash
+      run: |
+        mkdir -p ~/.aws
+        echo '${{ inputs.AWS_CONFIG }}' > ~/.aws/config
+      
+        aws configure set source_profile default --profile ${{ inputs.AWS_PROFILE }}
+        sed -i -z 's/credential_source = Environment\n//g' ~/.aws/config
+        echo '${{ inputs.AWS_ACCESS_KEY_ID }}
+        ${{ inputs.AWS_SECRET_ACCESS_KEY }}
+        ${{ inputs.AWS_REGION }}
+        
+        ' | aws configure --profile default
+
+    - name: Checkout integration-infra repo
+      uses: actions/checkout@v3
+      with:
+        repository: espoon-voltti/integration-infra
+        token: ${{ inputs.MVN_REPO_TOKEN }}
+
+    - name: Apply new build to kubernetes
+      shell: bash
+      run: |
+        cd kubernetes/${{ inputs.NAME }}
+        aws eks update-kubeconfig --profile ${{ inputs.AWS_PROFILE }} --name ${{ inputs.CLUSTER_NAME }}
+        ../../bin/eks-apply.sh
```

Comment:
Where's kubectl installed and it's version defined? It should be easy to change when cluster version updates are done